python/dask_cudf/dask_cudf/core.py

```diff
@@ -374,62 +374,47 @@ def last_index(df):
         divisions = compute(*divs)
         return type(self)(self.dask, self._name, self._meta, divisions)
 
-    def set_index(self, index, drop=True, sorted=False):
+    def set_index(
+        self,
+        other,
+        drop=True,
+        sorted=False,
+        npartitions=None,
+        divisions=None,
+        inplace=False,
+        **kwargs,
+    ):
         """Set new index.
 
-        Parameters
-        ----------
-        index : str or Series
-            If a ``str`` is provided, it is used as the name of the
-            column to be made into the index.
-            If a ``Series`` is provided, it is used as the new index
-        drop : bool
-            Whether the first original index column is dropped.
-        sorted : bool
-            Whether the new index column is already sorted.
+        See: dd.core.DataFrame.set_index
```

Comment:
I think that if you omit a docstring in python that it defers to the docstring of the method on the superclass.  So we may want to remove all of this.